ui/js/models/resultsets_store.js

```diff
@@ -143,11 +144,10 @@ treeherder.factory('ThResultSetStore', [
                     _.pick($location.search(), ["exclusion_profile", "visibility"]));
             }
             lastPolltime = Date.now();
-
             jobUpdatesPromise.then(function(jobList) {
+                jobList = _.flatten(jobList);
                 if (jobList.length > 0) {
                     lastJobUpdate = getLastModifiedJobTime(jobList);
```

Comment:
It seems like it would be worthwhile to safeguard ``getLastModifeidJobTime`` so that it can't concat 'Z' onto ``undefined`` again, regardless of how it's used.  Instead of ``job.last_modified + 'Z'``  it would be better to have some sane default in case we can't get ``job.last_modified``.  Perhaps::

    last_modified = job.last_modified || "1995-12-01T00:00:00"  // arbitrary, but guaranteed to not be the max
    return new Date(last_modified + 'Z')